safe_transaction_service/history/migrations/0068_alter_multisigtransaction_origin.py

```diff
@@ -22,16 +22,15 @@ def prepare_migration_charfield_to_jsonfield(apps, schema_editor):
 def repair_backward_migration(apps, schema_editor):
     MultisigTransaction = apps.get_model("history", "MultisigTransaction")
     # Empty objects should be None
-    MultisigTransaction.objects.filter(origin__exact="{}").update(origin=None)
+    MultisigTransaction.objects.filter(origin="{}").update(origin=None)
```

Comment:
As this query might be slow, better do it together:

`MultisigTransaction.objects.filter(Q(origin="{}") | Q(origin='"{}"')).update(origin=None)`

Reply:
of course